platform-sdk/platform-apps/tests/ConsistencyTestingTool/src/main/java/com/swirlds/demo/consistency/ConsistencyTestingToolState.java

```diff
@@ -269,5 +279,8 @@ public void handleConsensusRound(final @NonNull Round round, final @NonNull Plat
         stateLong = NonCryptographicHashing.hash64(stateLong, round.getRoundNum());
 
         transactionHandlingHistory.processRound(ConsistencyTestingToolRound.fromRound(round, stateLong));
+
+        setChild(ROUND_HANDLED_INDEX, new StringLeaf(Long.toString(roundsHandled)));
+        setChild(STATE_LONG_INDEX, new StringLeaf(Long.toString(stateLong)));
```

Comment:
Is there a reason not to use the PlatformState here instead of introducing these two particular leaves?

Reply:
If I understand the idea of this class correctly, these fields are directly correspond to anything what the platform state has. Previously it was stored separately, so I decided to follow the pattern.

Reply:
Okay. But my point is that the platform state encapsulates this data in a dedicated PlatfromState singleton rather than maintains it in separate tree leaves. So I'm just wondering why do we have to maintain these leaves explicitly here instead of simply adding the regular PlatformState to the state that this particular tool uses.

The reason I'm asking this is I suspect I'll have to add support for RosterService data into this demo state as well. And I see that unlike other demo states, this one doesn't follow the regular pattern of creating a PlatformState singleton and stuff. And it's unclear why it doesn't do so, and how am I supposed to add support for RosterService to it in the future...

Reply:
> But my point is that the platform state encapsulates this data in a dedicated PlatfromState singleton rather than maintains it in separate tree leaves.

This is true but these values are _not_  a part of the state in production (unless I'm missing something). I can _repurpose_ some fields of `com.hedera.hapi.platform.state.PlatformState` object to store these values, but that would look even more confusing. I don' think it's a good idea. 

An alternative to this implementation would be to have a service defined with a corresponding protobuf object. That would fit the existing pattern for services better, but, at the same time, it would be an overkill for this particular case. I don't want to pollute the protocol with test objects.

Reply:
Oh, I didn't realized these particular two values aren't a part of the actual production PlatformState object. Thanks for clarifying this! It would be great to have a comment added in code here to clarify this as well, so as to remove any potential confusion for future readers of this code.

But that's no longer an issue for me. Please feel free to resolve the conversation. Thanks!